tests/api/views/test_accounting.py

```diff
@@ -71,6 +71,68 @@
+    ['long_term_resident_card', 'carte_resident_longue_duree'],
+    ['cire_unlimited_stay_foreigner_card', 'carte_cire_sejour_illimite_etranger'],
+    ['ue_family_member_residence_card', 'carte_sejour_membre_ue'],
+    ['ue_family_member_permanent_residence_card', 'carte_sejour_permanent_membre_ue'],
+    ['refugee_a_b_card', 'carte_a_b_refugie'],
+    ['refugees_stateless_annex_25_26', 'annexe_25_26_refugies_apatrides'],
+    ['registration_certificate', 'attestation_immatriculation'],
+    ['a_b_card', 'carte_a_b'],
+    ['subsidiary_protection_decision', 'decision_protection_subsidiaire'],
+    ['temporary_protection_decision', 'decision_protection_temporaire'],
+    ['professional_3_month_residence_permit', 'titre_sejour_3_mois_professionel'],
+    ['salary_slips', 'fiches_remuneration'],
+    ['replacement_3_month_residence_permit', 'titre_sejour_3_mois_remplacement'],
+    ['unemployment_benefit_pension_compensation_proof', 'preuve_allocations_chomage_pension_indemnite'],
+    ['cpas_certificate', 'attestation_cpas'],
+    ['household_composition_or_birth_certificate', 'composition_menage_acte_naissance'],
+    ['tutorship_act', 'acte_tutelle'],
+    ['household_composition_or_marriage_certificate', 'composition_menage_acte_mariage'],
+    ['legal_cohabitation_certificate', 'attestation_cohabitation_legale'],
+    ['parent_identity_card', 'carte_identite_parent'],
+    ['parent_long_term_residence_permit', 'titre_sejour_longue_duree_parent'],
+    [
+        'parent_refugees_stateless_annex_25_26_or_protection_decision',
+        'annexe_25_26_refugies_apatrides_decision_protection_parent',
+    ],
+    ['parent_3_month_residence_permit', 'titre_sejour_3_mois_parent'],
+    ['parent_salary_slips', 'fiches_remuneration_parent'],
+    ['parent_cpas_certificate', 'attestation_cpas_parent'],
+    ['cfwb_scholarship_decision', 'decision_bourse_cfwb'],
+    ['scholarship_certificate', 'attestation_boursier'],
+    ['ue_long_term_stay_identity_document', 'titre_identite_sejour_longue_duree_ue'],
+    ['belgium_residence_permit', 'titre_sejour_belgique'],
+]
+
+general_file_fields = doctorate_file_fields + [
+    ['staff_child_certificate', 'attestation_enfant_personnel'],
+]
+
+doctorate_original_file_uuids = {}
+doctorate_dto_file_uuids = {}
+doctorate_new_file_uuids = {}
+
+for field in doctorate_file_fields:
+    original_uuid = uuid.uuid4()
+    doctorate_original_file_uuids[field[0]] = [original_uuid]
+    doctorate_dto_file_uuids[field[1]] = [str(original_uuid)]
+    doctorate_new_file_uuids[field[1]] = [str(uuid.uuid4())]
+
+
+general_original_file_uuids = {}
+general_dto_file_uuids = {}
+general_new_file_uuids = {}
+
+for field in general_file_fields:
+    original_uuid = uuid.uuid4()
+    general_original_file_uuids[field[0]] = [original_uuid]
+    general_dto_file_uuids[field[1]] = [str(original_uuid)]
+    general_new_file_uuids[field[1]] = [str(uuid.uuid4())]
```

Comment:
J'aurais bien gardé ça dans la classe, car ce code est executé dès que tu importe le fichier (ce qui peut ralentir le lancement des tests, même quand le tests n'est pas executé)

Reply:
D'accord merci, bien noté, je modifie ça.